ml_api/lib/backend_darknet.py

```diff
@@ -19,11 +19,17 @@ class BOX(Structure):
 class DETECTION(Structure):
     _fields_ = [("bbox", BOX),
                 ("classes", c_int),
+                ("best_class_idx", c_int),
                 ("prob", POINTER(c_float)),
                 ("mask", POINTER(c_float)),
                 ("objectness", c_float),
-                ("sort_class", c_int)]
-
+                ("sort_class", c_int),
+                ("uc", POINTER(c_float)),
```

Comment:
Can you provide additional info for this change? I don't know what kind of regression testing I need to run to test this part.

Reply:
This code is now using latest Darknet version and its interface has changed. You can see it here: https://github.com/AlexeyAB/darknet/blob/master/darknet.py